charts/kubewarden-controller/values.yaml

```diff
@@ -22,17 +22,28 @@ policyServer:
     repository: ghcr.io/kubewarden/policy-server
     tag: "v0.2.1"
   serviceAccountName: policy-server
-# All permissions are cluster-wide. Even namespaced resources are
-# granted access in all namespaces at this time.
+  # All permissions are cluster-wide. Even namespaced resources are
+  # granted access in all namespaces at this time.
   permissions:
-  - apiGroup: ""
-    resources:
-    - namespaces
-    - services
-  - apiGroup: "networking.k8s.io"
-    resources:
-    - ingresses
-  env: []
+    - apiGroup: ""
+      resources:
+        - namespaces
+        - services
+    - apiGroup: "networking.k8s.io"
+      resources:
+        - ingresses
+  telemetry:
+    enabled: True
```

Comment:
do we want it enabled by default? If I understood correctly, we would depend on the collector present then, which I think it's too much work for new users.
Also, upgrading charts would now depend on the collector, so we need a release note at least.

Reply:
I think we want telemetry enabled by default, but disabling it on the very first quick start of the docs.

Reply:
I prefer to have that disabled by default too. Not everybody reads the docs...

Reply:
Okay, I'm going to make it disabled by default then and remove the change on the quick start.

Reply:
Updated, and https://github.com/kubewarden/docs/pull/65 updated too.